From: vidplace7 <vidplace7@gmail.com>
Subject: [PATCH] Remove include of /usr/include in platformio.ini

Removes include of /usr/include in the platformio.ini file.

Signed-off-by: vidplace7 <vidplace7@gmail.com>
diff -ruN a/variants/portduino/platformio.ini b/variants/portduino/platformio.ini
--- a/variants/portduino/platformio.ini
+++ b/variants/portduino/platformio.ini
@@ -3,7 +3,6 @@ extends = portduino_base
 ; The pkg-config commands below optionally add link flags.
 ; the || : is just a "or run the null command" to avoid returning an error code
 build_flags = ${portduino_base.build_flags} -O0 -I variants/portduino
-  -I /usr/include
   !pkg-config --libs libulfius --silence-errors || :
   !pkg-config --libs openssl --silence-errors || :
 board = cross_platform
